provider/docker/labels.go

```diff
@@ -42,21 +42,30 @@ func getFuncMapLabel(labelName string) func(container dockerData) map[string]str
 }
 
 func parseMapLabel(container dockerData, labelName string) map[string]string {
-	customHeaders := make(map[string]string)
-	if label, err := getLabel(container, labelName); err == nil {
-		for _, headers := range strings.Split(label, ",") {
-			pair := strings.Split(headers, ":")
+	if parts, err := getLabel(container, labelName); err == nil {
+		if len(parts) == 0 {
+			log.Errorf("Could not load %q", labelName)
+			return nil
+		}
+
+		values := make(map[string]string)
+		for _, headers := range strings.Split(parts, "🧀") {
```

Comment:
Since we are splitting the headers in the label on this emoji, are we going to get users to separate custom headers etc on 🧀  ?

`X-Custom-Header:foo:cheese:X-Other-Header:bar`?

Reply:
It's a joke! 😂